packages/frontend/scss/message/_message.scss

```
@@ -121,9 +121,43 @@
         // firefox does not support this, but chromium and webkit do https://caniuse.com/?search=user-drag
         -webkit-user-drag: none;
       }
-      div.name {
+      div.info-text {
+        display: flex;
+        flex-direction: row;
+        max-width: 400px;
+        min-width: 0;
+        white-space: nowrap;
         font-size: medium;
         font-weight: 300;
+        gap: 3px;
+      }
+      div.name {
+        flex-shrink: 0;
+        flex-grow: 1;
+        min-width: 0;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+      }
+      div.document {
+        flex-shrink: 1;
```

```suggestion
      div.name {
        min-width: 0;
        overflow: hidden;
        text-overflow: ellipsis;
        white-space: nowrap;
      }
      div.document {
        flex-shrink: 999999999999999;
```

This removes `flex-shrink: 0;` from name, such that it never overflows the message bubble (e.g. if the name is super long), while making sure that "document" shrinks before the name.